samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml

```diff
@@ -0,0 +1,163 @@
+metadata:
+  version: "1"
+  dependencies:
+    apps:
+      - id: dynatrace.automations
+        version: ^1.728.0
+      - id: dynatrace.jira
+        version: ^4.1.1
```

Comment:
same: version is not released, but at least everything seems to work after importing the WF in prod